### PR TITLE
[FIX] web, *: adapt colors for dark mode

### PR DESCRIPTION
**Pull request by Brieuc-brd**
This PR fixes and adapts some color issues for the dark mode.

Requires:
- https://github.com/odoo/enterprise/pull/32657

task-2710677

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Brieuc-brd** [comment]
@stefanorigano 👋

File: addons/survey/static/src/scss/survey_survey_views.dark.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// = Survey view
+// ============================================================================
+// No CSS hacks, variables overrides only
+
+.o_survey_form {
+    --SurveyForm__IsSection: #{$o-gray-300};
```
Comment (stefanorigano) [inline addons/survey/static/src/scss/survey_survey_views.dark.scss:6]: makes sense? 

```suggestion
    --SurveyForm__section-bg: #{$o-gray-300};
```

Comment (Brieuc-brd) [reply addons/survey/static/src/scss/survey_survey_views.dark.scss]: Fixed 👍

**Comment by stefanorigano** [comment]
Since dark-mode is already deployed, all these commits are [FIX] ;)

**Review by adr-odoo — APPROVED** [review]
@robodoo rebase-ff r+

**Comment by adr-odoo** [comment]
@robodoo r+

**Pull request closed without merging**